client/src/components/Header/index.jsx

```diff
@@ -53,7 +54,7 @@ const Header = () => {
         </MenuItem>
         <HeaderMenuRight>
           <MenuItem>
-            <SearchBox />
+            <SearchBox disabledSearchBox={Boolean(disabledSearchBox)} />
```

Comment:
`disabledSearchBox`  is already of type boolean. You don't need to coerce it into Boolean again

Reply:
done